### ISSUE TITLE
Поддержка Dependency Injection 

### ISSUE THREAD

**Issue by pfedotovsky** [DotNetRu/App#71]
Сейчас текущие сервисы (получение митапов, друзей, докладов) являются static классами.

Нужно встроить DI, e.g. https://xamarinhelp.com/xamarin-forms-dependency-injection/

**Comment by vadlit** [comment]
Похоже, задача не очень актуальна? Или хочется сервис RealmService сделать инстанцируемым и доставать его из DI ?

**Comment by pfedotovsky** [comment]
@vadlit, да, хочется сделать сервисы инстанциируемыми. Но не уверен, насколько это нужно :) Попробовать можно.

**Comment by vadlit** [comment]
Звучит как переделка ради переделки )
Если бы сервисов действительно было несколько, или хотя бы у него был бы какой-то стейт, который можно хранить в полях класса, то было бы актуально, а так не очень понятно зачем..

**Comment by pfedotovsky** [comment]
@vadlit согласен. Если интересно поиграться с разными DI, можно сделать. Хуже, думаю не станет.
